packages/core/src/config/types.ts

```diff
@@ -142,17 +145,22 @@ export type SphinxConfig = {
   saltNonce: string
 }
 
-export interface RawCreate2ActionInput extends SphinxTransaction {
+export enum ActionInputType {
+  CREATE,
+  CREATE2,
```

Comment:
The main benefit of having an `ActionInputType` for `CREATE2` is that it lets us display these deployments in a readable format in the preview. Otherwise, it'd look something like this:
```
(0x4e59...).call(0x00000....)
```

This is currently what `CREATE3` deployments look like, which is why I actually think we should have an `ActionInputType` for these types of deployments too

Reply:
I understand these are breaking changes and it's not clear how we could scale this pattern to many different action types, but my main point is that they serve a purpose